server/src/routes/api/event.mjs

```diff
@@ -26,17 +26,19 @@ import * as events from '../../events.mjs';
 
 // POST /api/v1/event
 // Expected body: { method: 'device.onDeviceNameChanged' result: ... }
-function sendEvent(req, res) {
+function sendEvent(req, res, sendSuccessFlag = true) {
   const { ws } = res.locals; // Like magic!
   const { method, result } = req.body;
 
   function fSuccess() {
```

Comment:
All of the fSuccess, etc. stuff only exists in the event case because we had the weird case where we are reusing the events.sendEvent function between here and the ctx.sendEvent helper function. We don't need that stuff here. This function should basically just be the HTTP-related stuff (get body, parse params if any (no), etc.; eventually return 200 or 400 or whatever). But in the middle it should just call a function like sequence.sendSequence. Nice and simple.